collection_2020-03-24.sql

```diff
@@ -0,0 +1,46 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.29)
+# Database: collection
+# Generation Time: 2020-03-24 09:46:43 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table roadbikes
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `roadbikes`;
+
+CREATE TABLE `roadbikes` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `model` varchar(11) DEFAULT NULL,
```

Comment:
Well done for setting field lengths here, but will 11 characters be enough for the model/ brand?

Reply:
I think it's safe for brand, but model could be longer. Honestly the field lengths are just the defaults.